com.synology.SynologyDrive.yaml

```diff
@@ -51,6 +52,7 @@ modules:
       - type: extra-data
         filename: synology-drive.deb
         only-arches: [x86_64]
-        url: https://global.download.synology.com/download/Tools/SynologyDriveClient/2.0.2-11078/Ubuntu/Installer/x86_64/synology-drive-client-11078.x86_64.deb
+        url: http://192.168.2.2/synology-drive.deb
```

Comment:
that doesn't look like the correct url

Reply:
i know and that's why i already pushed a commit (that correct the typo)
i've a very slow internet connection and that's my workaround when i run "flatpak-builder _blablablabla_"